drivers/usb/serial/xr_usb_serial_hal.c

```diff
@@ -0,0 +1,855 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+ */
+
+#define XR_SET_MAP_XR2280X		5
+#define XR_GET_MAP_XR2280X		5
+
+#define XR_SET_MAP_XR21B142X		0
+#define XR_GET_MAP_XR21B142X		0
+
+#define XR_SET_MAP_XR21V141X		0
+#define XR_GET_MAP_XR21V141X		1
+
+#define XR_SET_MAP_XR21B1411		0
+#define XR_GET_MAP_XR21B1411		1
+
+
+int xr_usb_serial_set_reg(struct xr_usb_serial *xr_usb_serial,int regnum, int value)
+{
+	int result;
+	int channel = 0;
+	//dev_info(&xr_usb_serial->control->dev, "%s Channel:%d 0x%02x = 0x%02x\n", __func__,channel,regnum, value);
```

Comment:
you can use `dev_dbg(..)` here.... or if this line it's not needed, just remove it....

Reply:
@rene this is the code from the original code. I did not want to touch it in case we ever need to updat to a new version